.svgrrc.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  template: require('./svgr-template'),
+  dimensions: false,
+  replaceAttrValues: {
+    '#FF0000': '{props.fill}',
+    '#ff0000': '{props.fill}',
+    red: '{props.fill}',
+    '#000': '{props.fill}',
+    '#f00': '{props.fillSecondary}'
```

Comment:
it looks like this original intent was to have the red part of the svg have the secondary fill so there might be some inconsistencies here. Do the other versions of red only occur on web?

Reply:
oh i see what you mean... so the fillSecondary is only there to service the legacy mobile svgs anyway, but i see what you mean how this may cause confusion... once we move all icons to harmony i think we can drop fillSecondary